ui/app/src/state/actions/preview.ts

```diff
@@ -175,4 +179,31 @@ export function setItemBeingDragged(active: boolean): StandardAction {
   };
 }
 
+export const fetchAudiencesPanel = createAction(FETCH_AUDIENCES_PANEL);
```

Comment:
You're not fetching the panel. You're fetching it's config. Name semantically...

FETCH_AUDIENCES_PANEL_FORM_DEFINITION, etc

Reply:
@rart done, need to discuss if setActiveProfile should go somewhere else than preview

Reply:
Leave a TODO 🙂